app/assets/stylesheets/pages/_lesson.scss

```diff
@@ -1 +1,81 @@
+.tab-content {
+  width: 45rem;
+  height: auto;
+  border-radius: 2rem;
+  background-color: #052c65;
+}
 
+.filter-icons {
+  width: 3.5rem;
+  height: auto;
+  border-radius: 50%;
+  box-shadow: 10px 10px 18px -6px rgba(0,0,0,0.75);
+}
+
+.nav-tabs {
+  width: 7rem;
+  height: 20rem;
+  display: flex;
+  flex-direction: column;
+  align-self: center;
+}
+
+.header {
```

Comment:
This class naming convention might be too generic and effect other pages, unless we are using it as a standard for all headers. Just something to keep in mind in case for other pages